src/main/java/seedu/duke/util/Parser.java

```diff
@@ -63,10 +65,10 @@ public Command parseCommand(String userInput) {
 
         try {
             switch (commandString.toLowerCase()) {
-            case AddCommand.COMMAND_WORD_NOTE:
+            case AddNoteCommand.COMMAND_WORD_NOTE:
                 return prepareAddNote(userMessage);
-            case AddCommand.COMMAND_WORD_EVENT:
-                // return prepareAddEvent(userMessage);
+            /*case AddNoteCommand.COMMAND_WORD_EVENT:
```

Comment:
Remind the person who is working on AddEvent not to forget this part! 👍

Reply:
Already informed